### ISSUE TITLE
list item cannot have aria selected on certain roles

### ISSUE THREAD

**Issue by e111077** [material-components/material-components-web-components#1271]
**Describe the bug**

on specific roles, aira-selected should not be set on list item. e.g. navigation menu

**To Reproduce**

1. set role incompatible with aria-selected on list item
2. select the list item
3. run a11y audit

**Expected behavior**

should pass a11y audit

**Browser Version (please complete the following information):**
 - OS: mac os
 - Browser audit on chrome
